uniparc-rest/src/main/java/org/uniprot/api/uniparc/controller/UniParcController.java

```diff
@@ -334,6 +337,7 @@ public ResponseEntity<MessageConverterContext<UniParcEntry>> searchByDBRefId(
     @Operation(
             hidden = true,
             summary = PROTEOME_UPID_UNIPARC_OPERATION,
+            description = PROTEOME_UPID_UNIPARC_OPERATION_DESC,
             responses = {
                 @ApiResponse(
                         content = {
```

Comment:
description is missing for sequenceFilter, getByUpis GET and getByUpis POST endpoints